libraries/qr_make.py

```diff
@@ -0,0 +1,15 @@
+import qrcode
+
+def qr_make(web_link, CHAT_ID):
+    qr = qrcode.QRCode(
+    version=6,
+    error_correction=qrcode.constants.ERROR_CORRECT_H,
+    box_size=10,
+    border=4,
+)
+
+    qr.add_data(web_link)
+    qr.make(fit=True)
+    img = qr.make_image(fill_color="black",
+                    back_color='white').convert('RGB')
+    img.save('test_' + CHAT_ID + '.png')
```

Comment:
I think it is better to split image creation and image saving into different methods.

Reply:
Updated this, by creating two separate methods in lib